packages/flutter/lib/src/services/raw_keyboard_macos.dart

```diff
@@ -0,0 +1,315 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'keyboard_key.dart';
+import 'keyboard_maps.dart';
+import 'raw_keyboard.dart';
+
+/// Platform-specific key event data for macOS.
+///
+/// This object contains information about key events obtained from macOS's
+/// `NSEvent` interface.
+///
+/// See also:
+///
+///  * [RawKeyboard], which uses this interface to expose key data.
+class RawKeyEventDataMacOs extends RawKeyEventData {
+  /// Creates a key event data structure specific for macOS.
+  ///
+  /// The [characters], [charactersIgnoringModifiers], and [modifiers], arguments
+  /// must not be null.
+  const RawKeyEventDataMacOs(
+      {this.characters = '',
+      this.charactersIgnoringModifiers = '',
+      this.keyCode = 0,
+      this.modifiers = 0})
+      : assert(characters != null),
```

Comment:
The colon before initializers/asserts should be on the same line as the closing parenthesis.